src/go/rpk/pkg/cli/cmd/cluster/partitions/status.go

```diff
@@ -0,0 +1,79 @@
+package partitions
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewBalancerStatusCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "balancer-status",
+		Short: "Queries cluster for partition balancer status",
+		Long: `Queries cluster for partition balancer status:
+
+If continuous partition balancing is enabled, redpanda will continuously
+reassign partitions from unavailable nodes and from nodes using more disk space
+than configured limit.
+
+This command can be used to monitor the partition balancer status.
+
+FIELDS
+
+    Status:                        Either off, ready, starting, in_progress or stalled.
+    Seconds Since Last Tick:       The last time the partition balancer ran.
+    Current Reassignments Count:   Current number of partition reassignments in progress.
+    Unavailable Nodes:             The nodes that have been unavailable after a time set
+                                   by the 'partition_autobalancing_node_availability_timeout_sec' 
+                                   cluster property.
+    Over Disk Limit Nodes:         The nodes that surpassed the threshold of used disk
+                                   percentage set by the 'partition_autobalancing_max_disk_usage_percent'
+                                   cluster property.
+
+BALANCER STATUS
+
+    off:          The balancer is disabled.
+    ready:        The balancer is active but there is nothing to do.
+    starting:     The balancer is starting but has not run yet.
+    in_progress:  The balancer is active and is in the process of scheduling partition movements.
+    stalled:      There are some violations, but for some reason, the balancer cannot make
```

Comment:
Hmmm...this is a tough one, because the doc is essentially telling the customer that something is wrong, but we have no idea what that something is. Understand this could likely have many causes. Still, we should offer up some action for the use to take--should they check the config? Contact Redpanda Support (<--typically, measure of last resort in docs).

Maybe:
Violations have been detected and the balancer cannot correct them. <then include action user can/should take>

Reply:
@ztlpn What action do you think the user can/should take after seeing `stalled` status in the balancer status?

Reply:
@Feediver1 should we add this to a section here, or should we refer to online docs?

Reply:
I think the user experience should be the top criteria, so would recommend putting the info in the message - this saves customers from having to jump from product over to docs to look it up. We should also put this guidance in the docs (filed Issue , but not require customers to jump to the docs to act on the stalled error msg.

Perhaps:
Violations have been detected and the balancer cannot correct them. Check the following:
- Are there are enough healthy nodes to which to move partitions? For example, in a 3-node cluster no movements are possible, so you will see a stall every time there is a violation.
- Does the cluster have sufficient space? Nodes with over 80% of used disk space cannot be rebalanced.
- Do all partitions have quorum? If two of the three partition replicas are down, this partition cannot be moved.
- Are any nodes in maintenance mode? Partitions are not moved if a node is in maintenance mode.

Reply:
@micheleRP

Reply:
> Nodes with over 80% of used disk space cannot be rebalanced.

Just to correct the phrasing, should be "if there are *no* nodes in the cluster with less than 80% of used disk space, rebalancing cannot proceed". Or "all nodes are with more than 80%". Not just a single node.

Reply:
I just added this troubleshooting guide :+1:

Reply:
I also added info about stalled status in the new Continuous Data Balancing doc: https://deploy-preview-470--redpanda-documentation.netlify.app/docs/core/cluster-administration/continuous-data-balancing/#use-data-balancing-commands

Reply:
@micheleRP We just updated the wording a little bit for the Stalled Status Troubleshooting. It might be worth changing this in the docs too. :D

Reply:
> @micheleRP We just updated the wording a little bit for the Stalled Status Troubleshooting. It might be worth changing this in the docs too. :D

Thanks, I've updated the info about the status command in the Continuous Data Balancing doc: https://deploy-preview-470--redpanda-documentation.netlify.app/docs/core/cluster-administration/continuous-data-balancing/#use-data-balancing-commands